src/components/icons/FreediIcon.tsx

```diff
@@ -0,0 +1,29 @@
+<path d="M74.7531 8.49805C74.7531 9.4681 74.5968 10.3796 74.2843 11.2324C73.9783 12.0853 73.5421 12.8275 72.9757 13.459C72.4093 14.0905 71.729 14.5885 70.9347 14.9531C70.1469 15.3177 69.2713 15.5 68.3077 15.5H61.2472V13.498H68.3077C68.9978 13.498 69.6163 13.3678 70.1632 13.1074C70.7166 12.847 71.1821 12.4922 71.5597 12.043C71.9438 11.5872 72.2368 11.0566 72.4386 10.4512C72.6469 9.83919 72.7511 9.18815 72.7511 8.49805C72.7511 7.80794 72.6469 7.16016 72.4386 6.55469C72.2368 5.94922 71.9438 5.42188 71.5597 4.97266C71.1821 4.51693 70.7166 4.15885 70.1632 3.89844C69.6163 3.63151 68.9978 3.49805 68.3077 3.49805H60.2511V15.5H58.2491V1.49609H68.3077C69.2713 1.49609 70.1469 1.68164 70.9347 2.05273C71.729 2.41732 72.4093 2.91862 72.9757 3.55664C73.5421 4.18815 73.9783 4.93034 74.2843 5.7832C74.5968 6.62956 74.7531 7.53451 74.7531 8.49805ZM79.5675 2.00391C79.5675 2.21224 79.5284 2.40755 79.4503 2.58984C79.3722 2.76562 79.2648 2.92188 79.1281 3.05859C78.9913 3.19531 78.8318 3.30273 78.6495 3.38086C78.4672 3.45898 78.2752 3.49805 78.0734 3.49805C77.865 3.49805 77.6697 3.45898 77.4874 3.38086C77.3051 3.30273 77.1456 3.19531 77.0089 3.05859C76.8722 2.92188 76.7648 2.76562 76.6866 2.58984C76.6085 2.40755 76.5695 2.21224 76.5695 2.00391C76.5695 1.80208 76.6085 1.61003 76.6866 1.42773C76.7648 1.24544 76.8722 1.08594 77.0089 0.949219C77.1456 0.8125 77.3051 0.705078 77.4874 0.626953C77.6697 0.542318 77.865 0.5 78.0734 0.5C78.2752 0.5 78.4672 0.542318 78.6495 0.626953C78.8318 0.705078 78.9913 0.8125 79.1281 0.949219C79.2648 1.08594 79.3722 1.24544 79.4503 1.42773C79.5284 1.61003 79.5675 1.80208 79.5675 2.00391ZM79.0695 15.5H77.0675V4.50391H79.0695V15.5Z" fill="#10C0AB"/>
+</svg>
+    );
+}
```

Comment:
Please use the same code format as in other files.

Reply:
done